src/app/@shell/ui/header/header.component.ts

```diff
@@ -87,6 +94,10 @@ export class HeaderComponent implements OnInit, OnDestroy {
   >(undefined);
   private subscriptionTransaction$?: Subscription;
   private subscriptionNotification$?: Subscription;
+  public cartItemCount = 0;
+  private cartItemsSubscription!: Subscription;
```

Comment:
Soonaverse use "$" to indicate something is subscription. Can you correct that? It makes understanding code much easier.

Reply:
You bet, will make all future Subscriptions follow that standard.